cluster-provision/gocli/cmd/nodesconfig/opts.go

```diff
@@ -207,3 +207,9 @@ func WithAAQVersion(aaqVersion string) K8sConfigFunc {
 		n.AAQVersion = aaqVersion
 	}
 }
+
+func WithDNC(dnc bool) K8sConfigFunc {
```

Comment:
Please consider moving this function closer to the other functions controlling CNAO.